mne/coreg.py

```diff
@@ -1207,3 +1207,56 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+def get_mni_fiducials(subject, subjects_dir=None):
+    """Estimate fiducials for a subject.
+
+    Parameters
+    ----------
+    subject : str
+        Name of the mri subject
+    subjects_dir : None | str
+        Override the SUBJECTS_DIR environment variable
+        (sys.environ['SUBJECTS_DIR'])
+
+    Returns
+    -------
+    fids_mri : list
+        List of estimated fiducials (each point in a dict)
+
+    Notes
+    -----
+    This takes the ``fsaverage-fiducials.fif`` file included with MNE—which
+    contain the LPA, nasion, and RPA for the ``fsaverage`` subject—and
+    transforms them to the given FreeSurfer subject's MRI space.
+    The MRI of ``fsaverage`` is already in MNI Talarach space, so applying
+    the inverse of the given subject's MNI Talairach affine transformation
+    (``$SUBJECTS_DIR/$SUBJECT/mri/transforms/talarach.xfm``) is used
+    to estimate the subject's fiducial locations.
+
+    For more details about the coordinate systems and transformations involved,
+    see https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems and
+    :ref:`plot_source_alignment`.
+    """
+    # Eventually we might want to allow using the MNI Talarach with-skull
+    # transformation rather than the standard brain-based MNI Talaranch
+    # transformation, and/or project the points onto the head surface
+    # (if available).
```

Comment:
@jhouck while writing a description of the process I had some ideas for other ways it could be refined, so I put them in the comments. Just in case we want to try to make it better someday...

Reply:
I think fsaverage doesn't ship with the talairach_with_skull.lta -- at least, none of mine has it -- but worth a try. If nothing else we could look at the relationship between the talairach and talairach_with_skull in a large-ish sample and gin something up based on that.